### ISSUE TITLE
Feature: Option to set mobile's buttons default scale size

### ISSUE THREAD

**Issue by shaylevary** [caiiiycuk/js-dos.com#261]
Hi,

I would be happy if it there will be an option to set a default buttons and nipple size (scale) in the mobile interface.
Currently the default button scale size is 1.8, which is a bit small and need to be resize to 2.5.
It would be nice if I could set a new default scale size on the running HTML script.

**Comment by caiiiycuk** [comment]
Fixed in v7.5.0, now you can set the scale/mirror and sensitivity in ctor like this:
```js
        Dos(document.getElementById("jsdos"), {
             scaleControls: 2.5,
             mirroredControls: true,
             sensitivityValue: 1,
```

**Comment by shaylevary** [comment]
Thank you very much, I really appreciate you for adding this feature.
BTW, When i set "mirroredControls: true," it display the nipple and arrow keys together on the right side, while when i don't use it, it appears ok (it display only the nipple on left side).
Is there any option to set it to display the nipple only without the arrow keys around?
![mirrored control](https://user-images.githubusercontent.com/10597517/212043409-38b4008f-d02b-4d04-9954-d3a63089a47c.jpg)
